include/highfive/h5easy_bits/H5Easy_xtensor.hpp

```diff
@@ -75,6 +67,22 @@ struct load_impl
     }
 };
 
+// front-end
+template <class T>
+inline DataSet dump(File& file,
+                    const std::string& path,
+                    T&& data,
+                    DumpMode mode)
+{
+    if (!file.exist(path)) {
+        return detail::xtensor::dump_impl(file, path, data);
```

Comment:
@alexsavulescu data is universal ref, why dont we std::forward(data)? This looks dangerous (copies may be made!)

Reply:
Actually, what was the prob with const-ref?

Reply:
We could indeed use `std::forward`, though for the moment there is not reason to prefer `T&& data` over `const T& data`. So I can change it to the latter if you prefer.

Reply:
Yes, we should go with const T& data.

Reply:
As you wish: https://github.com/BlueBrain/HighFive/pull/299